apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -268,6 +267,21 @@
         public static String PATTERN = "%level %timestamp %thread %class : %msg %throwable";
     }
 
+    public static class StatusCheck {
+        /**
+         * Ignored exception list, also affect their subclasses.
```

Comment:
```suggestion
         * Listed exceptions would not be treated as an error. Because in some codes, the exception is being used as a way of controlling business flow.
```